src/main/java/Duke.java

```diff
@@ -25,12 +56,27 @@ public static Boolean Exit(String args){
     }
      */
      public static boolean actionCommand(String args){
-         switch(args){
+         String command = filter(args);
+         int dividerPosition = args.indexOf(" ");
+         String taskNumber = args.substring(dividerPosition + 1, args.length());
+         switch(command){
              case "bye":
```

Comment:
I like the use of switch statements instead of if-else statements as your code is neat and clear